pkg/scheduler/scheduler.go

```diff
@@ -459,10 +460,6 @@ func (sched *Scheduler) preempt(state *framework.CycleState, fwk framework.Frame
 // This function modifies assumed if volume binding is required.
 func (sched *Scheduler) assumeVolumes(assumed *v1.Pod, host string) (allBound bool, err error) {
 	allBound, err = sched.VolumeBinder.Binder.AssumePodVolumes(assumed, host)
```

Comment:
```return sched.VolumeBinder.Binder.AssumePodVolumes(assumed, host)``` and remove the named return vars, or completely remove the function and call this directly in scheduleOne

Reply:
Call sched.VolumeBinder.Binder.AssumePodVolumes(assumed, host) directly and removed this function.